wasm2c/examples/rot13/rot13.c

```diff
@@ -0,0 +1,258 @@
+#include <math.h>
```

Comment:
I guess its obvious by it might be nice to have a comment at the top along the lines of `Automatically generated by wasm2c ...` (with the full command line maybe?)`

Reply:
Done.